### ISSUE TITLE
Database NULL's and 0's

### ISSUE THREAD

**Issue by Ivylia** [XxInvictus/FFXIV-Market-Calculator#12]
After running main.py and the log showing that it updated the items I checked the database file for my world, and while it has the names and number IDs of the items, all the other fields are NULL or 0.

I have all the requirements installed, and here is my config information. I had not run into this issue on previous versions. (Edit: I'm not sure why some of this is showing up in really big font, I'm sorry)

[MAIN]
# Set whether you want to scan a world or datacentre [World|Datacentre|Datacenter]
# Default: World
MarketboardType = World
# Set the Datacentre you wish to scan
# Default: Crystal
Datacentre = Primal
# Set the world you wish to scan
# Default: Zalera
World = Ultros
# Set how many results you want to be outputted
# Default: 50
ResultQuantity = 50
# Set how many items you want to refresh from Universalis (0 = ALL)
# Default: 0
UpdateQuantity = 0
# Set the threshold for Minimum Average Sales Per Day to include in results
# Default: 20
MinAvgSalesPerDay = 20
# Whether or not to also display results without Crafting Cost factor
# Default: False
DisplayWithoutCraftCost = True

[LOGGING]
# Whether or not to enable logging [True|False]
# Default: FALSE
LogEnable = True
# Set the logging level [CRITICAL|ERROR|WARNING|INFO|DEBUG]
# Default: Info
LogLevel = INFO
# Set the log open mode [WRITE|APPEND]
# Write will clear the log each run, Append will append to the end
# Warning Append could result in a VERY large log if not cleared or rotated outside
LogMode = WRITE
# Where to log the information relative to the python script directory
# Default: ffxiv_market_calculator.log
LogFile = ffxiv_market_calculator.log

[DISCORD]
# Discord functionality requires setting environment variables DISCORDID and DISCORDTOKEN to their respective values
# Whether to enable Discord Webhook
DiscordEnable = False
# Id's of the Discord messages to edit in a list format is [MessageId1, MessageId2...]
# 20 Results per discord message
# Having this as an empty list [] will create new messages
MessageIds = [123456789123456789,123456789123456789]

**Comment by XxInvictus** [comment]
TLDR, I have just done a major refactoring of how data is pulled, stored and displayed. Try and pull the latest version, bearing in mind.

I have also completely removed the minimum velocity option and changed the calculation to profit X velocity

**THE NEWEST VERSION WILL REQUIRE DESTROYING AND LETTING THE SCRIPT RECREATE THE DB's**
Added New columns so the databases need to be regenerated with them.

**Comment by XxInvictus** [comment]
I am gonna close this one out as there have been major changes around how data is pulled and stored which should fix this reported issue.
